### DISCUSSION TITLE
Étude de cas : "Traduction" P24

### DISCUSSION THREAD

**Discussion by Anto792** [Hypertopic/HyperGlosae#213] [Show and tell]
This video presenting all content we added to Hyperglosae during this semester.

https://github.com/Hypertopic/HyperGlosae/assets/70704575/a5a55159-19ab-4348-9e6d-dce1ee22698c

**Comment by PElbaz** [comment]
Explications claires.
J’ai eu quelques problèmes de son, notamment lors de la fonctionnalité « retour en arrière »

**Comment by benel** [comment]
@AliceGuib @koeltv @AgatheIV @anto792 @khbgbgk1 @PaulGitProject
Je vous transmets le retour de Diana Lemay : 

> Je viens de regarder la vidéo pour la traduction, je pense que c'est très bien ce que les étudiants ont pu améliorer même si j'avoue que je ne travaille qu'avec TraduXio et ne suis pas du tout familiarisée avec HyperGlosae.
> Merci pour ce partage et encore une fois, bravo aux étudiants.
> Est-ce qu'il y a des changements similaires prévus aussi pour TraduXio à l'avenir ? Le retour en arrière serait une fonctionnalité très utile par exemple. Si jamais il y a un groupe d'étudiants qui travailleront sur TraduXio l'année prochaine, je serai ravie de collaborer.
